I/ImageNoise/Compat.toml

```diff
@@ -0,0 +1,3 @@
+["0.1"]
+Distributions = "0.19"
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.